Update Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.i…

O-Mary opened the pull request:
This is an updated notebook as per feedback gotten.

ignaciomigliaro commented:
> This is an updated notebook as per feedback gotten.

Give better description of the changes, what did you change and what was the feedback.

O-Mary commented:
> > This is an updated notebook as per feedback gotten.
> 
> Give better description of the changes, what did you change and what was the feedback.

Whenever you run the initial code more than once, you continue to apply the symmetry operations until you get an error. So, I fixed that by moving the part where we deleted the first row of the matrix into the symmetry definitions and re-labeled it as atoms1. Also, I also added a function to return the resulting vector as a float.
